repobee_sanitizer/_sanitize_repo.py

```diff
@@ -93,7 +93,8 @@ def sanitize_files(
 
 
 def sanitize_to_target_branch(
-    repo_path: pathlib.Path, target_branch: str,
+    repo_path: pathlib.Path,
+    target_branch: str,
 ) -> List[_format.FileWithErrors]:
```

Comment:
Argh, these formatting changes were not actually necessary. Not your fault, though, it's a change in the behavior of black. Essentially, in the before time, black would allow trailing commas in parameter lists, and still keep tem on a single line (the deleted line here). Now, if black finds a trailing comma, it will always wrap the lines (the added lines here). The fix for that is to remove trailing commas from lines you don't want wrapped, which would have been the case here.

Just an FYI for the futere.